packages/terra-modal/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "react"],
+  "plugins": ["transform-object-assign", "transform-object-rest-spread", "transform-runtime"],
```

Comment:
@ryanthemanuel Don't worry. The config is migrated from the old module repo. Since terra-modal will depend on terra-overlay. I will close this PR and send a new one for modal after overlay is available in master.